spring-kafka/src/main/java/org/springframework/kafka/retrytopic/RetryTopicConfigurationBuilder.java

```diff
@@ -190,15 +254,37 @@ public RetryTopicConfigurationBuilder maxAttempts(int maxAttempts) {
 		return this;
 	}
 
+	/**
```

Comment:
Maybe we should add that this is in milliseconds? Had to go through the code to figure out 😄